ingestors/tabular/xlsx.py

```diff
@@ -48,6 +48,17 @@ def ingest(self, file_path, entity):
                 table = self.manager.make_entity("Table", parent=entity)
                 table.make_id(entity.id, name)
                 table.set("title", name)
+                # add workbook metadata to individual tables
+                for metadatum in [
+                    "authoredAt",
+                    "modifiedAt",
+                    "author",
+                    "summary",
+                    "generator",
+                    "language",
+                    "processingAgent",
+                ]:
+                    table.set(metadatum, entity.get(metadatum))
```

Comment:
In terms of DRY and conciseness: I wonder if we could move this code to the common `TableSupport` class. I see that all of these call [emit_row_tuples](https://github.com/alephdata/ingest-file/blob/1a77d9bee2591c776416252a87a2b3356b3e9b39/ingestors/support/table.py#L45), maybe that's a good place to put this?

Reply:
(If that is not somehow feasible then maybe extracting the common code to a function in `TableSupport` is a good compromise?)

Reply:
@stchris it also looks like an eyesore to me, I agree with you, but I kept it this way for 3 reasons:
1. the metadata that gets passed to children entities isn't the same between XLS/XLSX and ODS. So the "commonalities" exist only between XLS and XLSX.
2. the `emit_row_tuples` is also called by the CSV ingestor, which doesn't emit children entities, so I would end up complicating the logic of that method by checking is any children are going to be emitted or not.
3. the `TableSupport` class seems like it serves the limited purpose of handling rows and it doesn't have the concept of FTM hierarchies (or children entities). 

I would lean towards leaving the enumeration as-is, because it also keeps the metadata inheritance in a code area where the children are also emitted, making it easier to keep track of. To me, this made sense since the actual repetition only happens once, in the case of XLS & XLSX.